frontend/src/shared/components/tasque-button/button.component.sass

```diff
@@ -62,3 +62,18 @@
         background: none
         background-color: colorscheme(gray-400)
         cursor: not-allowed
+
+.red
+    background: #ff0000
```

Comment:
please use colorscheme, if the color is not in the scheme - add it to the scheme with a meaningful name

Reply:
Added additional colors to colorscheme. And used them instead of custom. Fixed.